content/hi/docs/reference/glossary/affinity.md

```diff
@@ -0,0 +1,22 @@
+---
+title: आत्मीयता
+id: affinity
+date: 2019-01-11
+full_link: /docs/concepts/scheduling-eviction/assign-pod-node/#affinity-and-anti-affinity
+short_description: >
+     पॉड्स को कहां रखा जाए, यह निर्धारित करने के लिए शेड्यूलर द्वारा उपयोग किए जाने वाले नियम
+aka:
+tags:
+- fundamental
+---
+
+कुबेरनेट्स में, _affinity_ नियमों का एक सेट है जो शेड्यूलर को संकेत देता है कि पॉड्स को कहाँ रखा जाए।
+
+<!--more-->
+आत्मीयता दो प्रकार की होती है:
+* [नोड आत्मीयता](/docs/concepts/scheduling-eviction/assign-pod-node/#node-affinity)
+* [पॉड-टू-पॉड आत्मीयता](/docs/concepts/scheduling-eviction/assign-pod-node/#inter-pod-affinity-and-anti-affinity)
+
+कुबेरनेट्स का उपयोग करके नियमों को परिभाषित किया गया है {{< glossary_tooltip term_id="label" text="labels">}},
+और {{< glossary_tooltip term_id="selector" text="selectors">}} में निर्दिष्ट किया {{< glossary_tooltip term_id="pod" text="pods" >}}, 
+और उन्हें या तो आवश्यक या पसंद किया जा सकता है, यह इस बात पर निर्भर करता है कि आप शेड्यूलर को कितनी सख्ती से लागू करना चाहते हैं।
```

Comment:
```suggestion
नियमों को कुबेरनेट्स {{< glossary_tooltip term_id="label" text="लेबल">}} और {{< glossary_tooltip term_id="selector" text="सेलेक्टर">}} 
का उपयोग करके परिभाषित किया गया है, जो {{< glossary_tooltip term_id="pod" text="pods" >}} में निर्दिष्ट हैं , 
और उनका उपयोग इस बात पर निर्भर करता है कि आप शेड्यूलर को कितनी सख्ती से लागू करना चाहते हैं।
```

Reply:
@anubha-v-ardhan This sounds good. Thanks

Reply:
Do not see this addressed. Please address the suggestion provided by @anubha-v-ardhan since it is not reflected.

Reply:
Yes I have changed that. Let me push the changes again